cli/cdsctl/workflow_as_code.go

```diff
@@ -395,15 +401,15 @@ func workflowInitRun(c cli.Values) error {
 	}
 
 	if err := workflowTarReaderToFiles(dotCDS, tr, true, true); err != nil {
-		return err
+		return fmt.Errorf("error on workflowTarReaderToFiles:%s", err)
 	}
 
 	//Configure local git
 	if err := gitRepo.LocalConfigSet("cds", "workflow", name); err != nil {
-		return err
+		return fmt.Errorf("error on LocalConfigSet:%s", err)
```

Comment:
i prefer the standard git error returned by the lib. Naming the function is not very user friendly